### DISCUSSION TITLE
Release Process

### DISCUSSION THREAD

**Discussion by ryanlong1004** [ufs-community/uwtools#50] [Ideas]
Writing down how we want to do this.

# Release Process

1.  Run `bump2version` to bump `MAJOR, MINOR, OR PATCH`.
2.  Enter `git push --tags` for the final branch update.
3.  Create a pull request

**Comment by christinaholtNOAA** [comment]
This is no longer relevant. Please see https://github.com/ufs-community/uw-team/wiki/Unified-Workflow-Tools-Release-Procedure.
